sockets.js

```diff
@@ -142,7 +142,12 @@ module.exports = function (server, config) {
     }
 
     function clientsInRoom(name) {
-        return io.sockets.clients(name).length;
+        var room = io.sockets.adapter.rooms[name];
+        if (room && room.length) {
+            return room.length;
```

Comment:
This doesn't work for me:
room.length is always undefined.
You should use `Object.keys(room).length` instead of `room.length`